sdk/python/feast/infra/online_stores/datastore.py

```diff
@@ -75,8 +75,10 @@ class DatastoreOnlineStoreConfig(FeastConfigBaseModel):
 
 class DatastoreOnlineStore(OnlineStore):
     """
-    OnlineStore is an object used for all interaction between Feast and the service used for offline storage of
-    features.
+    Google Cloud Datastore implementation of the online store interface.
```

Comment:
for this and redis, can you link to the corresponding reference doc on what the data model looks like? e.g. https://github.com/feast-dev/feast/blob/v0.23-branch/docs/specs/online_store_format.md#google-datastore-online-store-format

Reply:
done